web/src/beta/features/Dashboard/ContentsContainer/Assets/AssetCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { FC } from "react";
+
+import { Asset } from "@reearth/beta/features/Assets/types";
+import { Icon, Typography } from "@reearth/beta/lib/reearth-ui";
+import { styled, useTheme } from "@reearth/services/theme";
+
+type ContentProps = {
+  asset: Asset;
+  icon?: "image" | "file" | "assetNoSupport";
+  selectedAssetId?: string;
+  onAssetSelect?: (assetId: string) => void;
+};
+
+export const AssetCard: FC<ContentProps> = ({ asset, icon, selectedAssetId, onAssetSelect }) => {
+  const theme = useTheme();
+
+  const renderContent = () => {
+    switch (icon) {
+      case "image":
+        return <AssetImage url={asset.url} />;
+      case "file":
+      case "assetNoSupport":
+        return (
+          <IconWrapper>
+            <Icon icon="fileFilled" color={theme.content.weak} size="large" />
+          </IconWrapper>
+        );
+      default:
+        return null;
+    }
+  };
+
+  return (
+    <CardWrapper
+      isSelected={selectedAssetId === asset.id}
+      onClick={() => onAssetSelect?.(asset.id)}>
+      {renderContent()}
+      <AssetName>
+        <Typography size="body">{asset.name}</Typography>
+      </AssetName>
+    </CardWrapper>
+  );
+};
+
+const CardWrapper = styled("div")<{ isSelected: boolean }>(({ theme, isSelected }) => ({
+  display: "flex",
+  flexDirection: "column",
+  border: `2px solid ${isSelected ? theme.select.main : "transparent"}`,
```

Comment:
Why do we have a 2px border? I didn't find it on design.

Reply:
Sorry i didn't mean it should not have a border, the key point is `2px`, i think in our current design usually is `1px` border.